### ISSUE TITLE
Mobile compatibility?

### ISSUE THREAD

**Issue by shruuub** [daledavies/jump#22]
Hello, I know thats probably a newbie question, but since i couldnt find something about it in the readme, but does this load on mobile browsers? Since i have the default setup, but the sites arent displaying, only the background and menu are.

**Comment by daledavies** [comment]
It should do yes, at least it does on mine (Android 12 with Chrome). Have you tried it on a desktop browser first?

**Comment by daledavies** [comment]
Could you provide some more info about your setup? Number of sites and tags, are you using openweather etc. Maybe a screenshot.

**Comment by shruuub** [comment]
> It should do yes, at least it does on mine (Android 12 with Chrome). Have you tried it on a desktop browser first?

Nope, since i dont have access to one right now, will probably test it in a few hours.

**Comment by shruuub** [comment]
> Could you provide some more info about your setup? Number of sites and tags, are you using openweather etc. Maybe a screenshot.

Uhm, i just noticed the site isnt loading, probably going to redeploy it soon, but yey, I am using OWM, 6 sites, as well as the default 3 tags.

**Comment by daledavies** [comment]
I suspect the issue isn't strictly with mobile devices but that something else may be afoot.

**Comment by shruuub** [comment]
> I suspect the issue isn't strictly with mobile devices but that something else may be afoot.
 yup, you're right, for some reason the icons, as well as clock/weather arent loading

**Comment by daledavies** [comment]
Yay! Glad you got it working 😄
